dataflow/gpu-examples/tensorflow-landsat/run.yaml

```diff
@@ -40,16 +40,16 @@ steps:
   args:
   - /pipeline/main.py
   - --output-path-prefix=$_OUTPUT_PATH
+  - --gpu-type=$_GPU_TYPE
+  - --gpu-count=$_GPU_COUNT
   - --runner=DataflowRunner
   - --project=$PROJECT_ID
   - --region=$_REGION
-  - --job_name=$_JOB_NAME
   - --temp_location=$_TEMP_LOCATION
-  - --worker_machine_type=custom-1-13312-ext
```

Comment:
As this needed a 13 Gig to 1 mem ratio before, maybe worth adding a memory min limit to the hint as well

Reply:
Yes, I actually added a `--resource_hints=min_ram=32GB`. I tried 16GB but I was still hitting out of memory errors. Something changed where it tried to load more images per worker I guess.

Reply:
I may be running 2 cores on the VM? I think we may need to set --experiments=no_use_multiple_sdk_containers

Reply:
It's already using it in line 53